package-lock.json

```diff
@@ -0,0 +1,1184 @@
+{
+  "name": "good-influx",
```

Comment:
I'm not familiar with `package-lock.json`.  How did you generate it?  `npm shrinkwrap`?  ASP?  Should it be version controlled?